lib/model-error.js

```diff
@@ -5,10 +5,17 @@ class ModelError extends Error {
 	static get codes() {
 
 		return {
-			DATABASE_CONFIG_NOT_FOUND: 1,
-			WRONG_CALLBACK: 2,
-			DRIVER_METHOD_NOT_IMPLEMENTED: 3,
-			INVALID_VALUE: 4
+			// Model
+			WRONG_CALLBACK: 1,
+			DRIVER_METHOD_NOT_IMPLEMENTED: 2,
+			INVALID_VALUE: 3,
+			// Database Dispatcher
+			INVALID_SETTINGS: 4, //  settings found with invalid format (not an object)
```

Comment:
:bulb: :question: 

Si bien ahora es parte de Model, pregunto si no sería mejor que el helper del  Database-Dispatcher tenga su propia clase de error, asi cuando salta de una es re claro que es eso, antes de fijarse en el mensaje o el codigo de error.

No pasa nada si no es posible . Es una idea nomas

Reply:
No lo veo tan crucial, el message del **Error** dice '_Invalid config file, should be an object_'